externalpost/src/main/java/com/example/externalpost/controller/PostController.java

```diff
@@ -27,11 +31,15 @@ public class PostController {
     //containerFactory 지정해주기
     @KafkaListener(topics = topicName, groupId="test", containerFactory = "kafkaListenerContainerFactory") //containerFactory중요
     public void kafkaconsumer(RequestInfo requestInfo){
-        PostDto postDto = postManager.getpost(requestInfo); //해당 요청에 대해 post 정보 추출
-        if (!postDto.getPostNumber().isEmpty()){ //응답 문제시 빈 PostDto객체이므로 not_found 리턴
-
-            postDbService.savePost(postDto);
+        Optional<PostDto> postDto = postManager.getpost(requestInfo);//해당 요청에 대해 post 정보 추출
```

Comment:
```suggestion
PostDto postDto = postManager.getpost(requestInfo).orElseThrow({Something});
log.info("postDbService.savePost실행됨:" + postDto.toString());
````
isPresent() 불필요,
현재 if depth 불필요.